test/draw/gpu/BlitTest.ooc

```diff
@@ -91,17 +91,73 @@ BlitTest: class extends Fixture {
 			this testOffset(this sourceImage, correctImage, IntVector2D new(4, 1))
 			correctImage free()
 		})
+		this add("Blit blend", func {
+			correctImage := RasterMonochrome fromAscii(
+				"< *X>
+				<XXXXXXXX       >
+				<X*    *X       >
+				<X*    *X       >
+				<X*   XXX       >
+				<X****XXXXXXXX  >
+				<XXXXXXXX   *X  >
+				<     X*    *X  >
+				<     X*   XXX  >
+				<     X****XXX  >
+				<     XXXXXXXX  >
+				<               >
+				<               >"
+			)
+			this testBlend(this sourceImage, correctImage, IntVector2D new(0, 0), IntVector2D new(5, 4), BlendMode Add, 0) // Bit exact
+			this testBlend(this sourceImage, correctImage, IntVector2D new(0, 0), IntVector2D new(5, 4), BlendMode White, 16) // Approximated
+			correctImage free()
+		})
+		this add("Blit fill", func {
+			correctImage := RasterMonochrome fromAscii(
+				"< *X>
+				<XXXXXXXX      >
+				<X*    *X      >
+				<X*  XXXXXXXX  >
+				<X*  X*    *X  >
+				<X***X*    *X  >
+				<XXXXX*   XXX  >
+				<    X****XXX  >
+				<    XXXXXXXX  >
+				<              >
+				<              >"
+			)
+			this testBlend(this sourceImage, correctImage, IntVector2D new(0, 0), IntVector2D new(4, 2), BlendMode Fill, 0) // Bit exact
+			correctImage free()
+		})
 	}
 	testOffset: func (source, expected: RasterMonochrome, offset: IntVector2D) {
+		this testOffsetMode(source, expected, offset, BlendMode Fill, 0) // Bit exact
+		this testOffsetMode(source, expected, offset, BlendMode Add, 0) // Bit exact
```

Comment:
This is just a clamped integer addition so no rounding errors possible there either.